external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala

```diff
@@ -299,7 +299,7 @@ private[kafka010] class KafkaMicroBatchStream(
       if (content(0) == 'v') {
         val indexOfNewLine = content.indexOf("\n")
         if (indexOfNewLine > 0) {
-          val version = parseVersion(content.substring(0, indexOfNewLine), VERSION)
+          parseVersion(content.substring(0, indexOfNewLine), VERSION)
```

Comment:
so it's not parse but validate?

Reply:
Yep, as a first step just deleted it to see whether it's needed and turned out it's throwing exception under some circumstances.

Reply:
BTW, this pattern exists outside of the Kafka connector as well.

Reply:
maybe we can rename it to `validateVersion`.

Reply:
OK, renaming.